network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1252,6 +1252,13 @@ def get_banner(self):
         SynchronizedField('slug'),
     ]
 
+    def get_language_code(self, request):
+        """Accepts a request. Returns a language code (string) if there is one. Falls back to English."""
+        default_language_code = 'en'
```

Comment:
grab this from the settings?  https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/settings.py#L413